### DISCUSSION TITLE
Noob here - Question about Ports

### DISCUSSION THREAD

**Discussion by Felixkrell** [mozilla/hubs#3713] [General Questions & Discussion] [answered]
Hi guys!

I am planning to use Hubs for my teaching and maybe even a small conference this year. When I open ubs in my universities WLAN, everything works except the mic. Noone seems to be able to speak sadly.

I wanted to ask if there is a list of Ports that need to be opened for this to work. I found some stuff here on github but I don't know how up to date the previous discussions are. Is there an list somewhere thats easy to read for non-programmers? I would need to write a short ticket to my IT department.

Thank you!

**Comment by robinkwilson** [comment, marked as answer]
List of ports:

Personal (1 server ports to open):
80 - dialog (will change to 8443 in the future)
443 - reticulum
5349 - turn
51610 - 60999 : webrtc ports

Multi-server:
80 - dialog (will change to 443 in the future)
443 - reticulum / on stream server for turn (5349 for Turn in the future)
51610 - 60999 : webrtc ports
